### DISCUSSION TITLE
Stacked Sliders behave differently with classic mod than how they do in stable

### DISCUSSION THREAD

**Discussion by WitherFlower** [ppy/osu#17667] [Q&A] [answered]
I noticed while playing on lazer (with classic mod) that something was off with sliders during gameplay.

Essentially, on stable, the second slider doesn't accept input until the first one is finished, while in lazer the second slider accepts input as soon as the sliderhead of the first slider has been clicked.

I managed to isolate the issue on a short beatmap with only two sliders. In both clips, I click both keys on the first slider then click one key on the second slider. Notice how stable ignores one of the first two hits, while lazer registers a sliderbreak on the second slider.

**Stable :** 

https://user-images.githubusercontent.com/52672543/161821612-e120670b-20e9-4026-8600-80afe1e9d134.mp4

**Lazer :**

https://user-images.githubusercontent.com/52672543/161821617-f0a03c09-c555-4da0-92e0-7990450ebc2a.mp4

I think this is the issue behind the "strange slider miss" mentioned here : https://github.com/ppy/osu/discussions/17362#discussioncomment-2433499

**Comment by frenzibyte** [comment]
This sounds like it should be fixed completely, not only on "Classic" mod.

**Reply by WitherFlower** [reply]
I don't know, this behavior makes sliderstreams very hard to hit in stable, so maybe that's not wanted in lazer nomod gameplay.

**Reply by Sublimelime** [reply]
I might be wrong in my understanding here but isn't this just caused by notelock? When you hit both keys on stable, it registers a normal hit for the first, and notelocks the second (ignores the tap) - then the followup hit is normal on the second.

On lazer due to notelock being toned down, it just registers the simultaneous tap as a normal hit on the first, and a super early one on the second (causing SB/miss). It allows you to miss basically.

Right? Or does notelock not apply to sliders on stable? Been a while since I played on stable.

**Reply by smoogipoo** [reply]
While this does look like a notelock issue, the OP has classic mod enabled which intends to replicate osu-stable's notelock. Will need more investigation.

**Comment by smoogipoo** [comment, marked as answer]
Going to make an issue for this so we don't lose track of it: https://github.com/ppy/osu/issues/17675
